packages/calypso-analytics/package.json

```diff
@@ -5,19 +5,19 @@
 	"homepage": "https://github.com/Automattic/wp-calypso",
 	"license": "GPL-2.0-or-later",
 	"author": "Automattic Inc.",
-	"sideEffects": false,
-	"main": "dist/cjs/index.js",
-	"module": "dist/esm/index.js",
+	"sideEffects": true,
```

Comment:
hmm ok so `sideEffects` is a flag controlling whether webpack can safely prune unused exported functions or not. If it's `true` than all functions will always be included in the bundle because webpack can't work out if they are necessary just for their "sideEffects". https://stackoverflow.com/questions/49160752/what-does-webpack-4-expect-from-a-package-with-sideeffects-false

Our only side effect is initializing `window._tkq` when the library is loaded; `client/lib/analytics/index.js` does in fact depend on this having happened.

I've actually gone and contained all usage of `window._tkq` inside a new `pushEventToTracksQueue()` function, and updated the dependent code in `client/lib/analytics/` to use this. 

So now the code still has the "Side Effect" of adding to the `window` object, but it doesn't have `sideEffects` because there are no implicit dependencies between functions :blush:.

There are other places in `apps/` where we use `window._tkq` but fortunately these places check to ensure that it's been initialized. I probably should update them to all use `pushEventToTracksQueue` but I don't want to introduce the dependency on `@automattic/calypso-analytics` to those apps right now 😋